_health-care/_js/components/personal-information/VAInformationSection.jsx

```diff
@@ -17,17 +17,29 @@ class VaInformationSection extends React.Component {
           <ErrorableCheckbox
               label="Are you VA Service Connected 50% to 100% Disabled?"
               checked={this.props.data.isVaServiceConnected}
+              required={!false}
```

Comment:
I understand your question now. The only rational for making this a required field would be somehow "force" the Vet to acknowledge that she'd seen the information. I'll remove the attribute.